src/program-editors2.bib

```diff
@@ -0,0 +1,56 @@
+@misc{cooke, title={A History of Accessibility at IBM}, url={https://www.afb.org/aw/5/2/14760}, journal={A History of Accessibility at IBM}, author={Cooke, Annemarie}}
```

Comment:
* put newlines between items in a bibtex entry
* in titles, you have to put capitalized items in curly braces, e.g. `{IBM}`, or LaTeX will sometimes change the capitalization